Foundation/FileManager.swift

```diff
@@ -120,18 +122,333 @@ open class FileManager : NSObject {
         return result
     }
     
+    private enum _SearchPathDomain {
+        case system
+        case local
+        case network
+        case user
+        
+        init?(_ domainMask: SearchPathDomainMask) {
+            if domainMask == .systemDomainMask {
+                self = .system; return
+            }
+            if domainMask == .localDomainMask {
```

Comment:
Why not use switch case instead?

Reply:
IIRC I tried, Swift complained, and I switched over without investigating.